src/apollo/queries.ts

```diff
@@ -687,6 +687,7 @@ export const SWAP_TRANSACTIONS = gql`
       first: 1000
       where: { pair_in: $allPairs, timestamp_gte: $lastTime }
       orderBy: timestamp
+      orderDirection: desc
```

Comment:
What does this have to do with pro mode

Reply:
This is to put recent transactions at first.